vite.config.js

```diff
@@ -0,0 +1,36 @@
+import { defineConfig } from 'vite';
+import path from 'path';
+
+const mode = process.env.NODE_ENV || 'development';
+const isDev = mode === 'development';
+
+export default defineConfig({
+  build: {
+    target: 'node12',
```

Comment:
suggestion: I'd maybe leave a comment here detailing that we should target the oldest version of node we support so someone doesn't think it needs to be in inverse in the future and swap to say 22.